docs/config.rst

```diff
@@ -15,12 +15,32 @@ and we are continuously adding more.
 
 .. _list of supported countries:
 
-A list of countries that we currently support:
+We currently support 20 out of 25 humanitarian response plan (HRP) countries:
```

Comment:
I think here should we just add a caveat that not all data is available for all countries? Cause people might read it and expect their country to be there.